src/ray_cast.cpp

```diff
@@ -59,20 +59,21 @@ sensor_msgs::LaserScan RayCast::Scan(cv::Point2f start, double yaw) {
 
```

Comment:
since this is a forked package, let's try to not have space/newline changes just so incase we pull upstream.